include/databaseFunctions.php

```diff
@@ -57,39 +46,25 @@ function getNumberPosts(){
 		$result = dbQuery("
 				SELECT COUNT(postID)
 				FROM posts
+				WHERE isPic=0
 			")->fetch();
-		return $result;
+		return $result['COUNT(postID)'];
```

Comment:
This works, but it seems a little clunky to have `Count(postID)` be the name of the index. You can use an alias in the SQL query to give this field a simpler name:

```
SELECT COUNT(postID) AS Count
FROM posts
WHERE isPic=0
```